charts/kube-prometheus-stack/values.yaml

```diff
@@ -1284,8 +1284,8 @@ kubeEtcd:
   ##
   service:
     enabled: true
-    port: 2379
-    targetPort: 2379
+    port: 2381
+    targetPort: 2381
```

Comment:
Maybe we should have some logic here using the right port depnedeing on the k8s version?
The values of the ports could be empty and only used when actualy set.

Reply:
A pretty good idea, maybe I'll give it a try :D

Reply:
Hey @monotek , I checked kubernetes git documentation and history (via blame), it seems that etcd metrics was always running on port 2381.
https://github.com/kubernetes/kubernetes/blob/96a32be18dfbd38ff5cb1df4d68528e2cf226958/cmd/kubeadm/app/constants/constants.go#L96 

I checked kubernetes git documentation and history (via blame), it seems that etcd metrics was always running on port 2381

Reply:
Not sure about that: https://etcd.io/docs/v3.1/op-guide/monitoring/

I don't have time to check further. Have only my mobile phone the next 4 weeks...

Reply:
just tested it on a kind cluster. it is localhost:2381/metrics